Bio/SeqIO/GfaIO.py

```diff
@@ -29,7 +42,68 @@ def _check_tags(seq, tags):
                 )
 
 
-def GfaIterator(source):
+def _tags_to_annotations(tags):
+    """Build an annotations dictionary from a list of tags (PRIVATE)."""
+    annotations = {}
+    for tag in tags:
+        parts = tag.split(":")
+        if len(parts) < 3:
+            raise ValueError(f"Segment line has invalid tag: {tag}")
+        annotations[parts[0]] = parts[2]
+
+        # Check type of the tag and raise warning on a mismatch. These RegExs
+        # are part of the 1.0 standard.
+        if parts[1] not in "AifZJHB":
+            warnings.warn(f"Tag has invalid type: {parts[2]}", BiopythonWarning)
```

Comment:
I think the warning should actually print the type {parts[1]} here not the value field {parts[2]}.

Reply:
Yep, thanks! I must have missed that while copying from the other tag type warnings.